content/posts/May-2020.md

```diff
@@ -4,6 +4,40 @@ date: 2020-05-06
 draft: false
 ---
 
+## Week 22 (May 25th - May 29th)
+
+### packit
+
+* [The problem with kerberos initialization](https://github.com/packit-service/packit/issues/837)
+  was fixed. ([#838])
+* We released a new version of the packit package. ([release 0.11.1])
+* We have been also working on some smaller issues and preparation steps for our future work.
+  ([#841], [#843], [#846], [#847])
+
+[#838]: https://github.com/packit-service/packit/pull/838
+
+[#841]: https://github.com/packit-service/packit/pull/841
+[#843]: https://github.com/packit-service/packit/pull/843
+[#846]: https://github.com/packit-service/packit/pull/846
+[#847]: https://github.com/packit-service/packit/pull/847
+[release 0.11.1]: https://github.com/packit-service/packit/releases/tag/0.11.0
+
+### packit-service
+
+* [Jirka] implemented creating new bug in Bugzilla for CentOS-Stream pull-requests ([#627]).
+  This will be triggered by the added label in the (near) future.
+* You will be able to use aliases also for koji builds. ([packit:#839], [#632])
```

Comment:
Use present tense (it's already possible, or shortly will be). Maybe give an example or link to docs with examples.

Reply:
- It's possible but I don't want to motivate users to do it. (It's in the "preview" mode in the docs.)

Reply:
> Maybe give an example or link to docs with examples.

We already use aliases for the other jobs. I don't think it's necessary here.